src/main/java/org/mule/extension/http/internal/HttpConnectorConstants.java

```diff
@@ -30,6 +30,7 @@
   String RETRY_ATTEMPTS_PROPERTY = SYSTEM_PROPERTY_PREFIX + "http.client.maxRetries";
   String RETRY_ON_ALL_METHODS_PROPERTY = SYSTEM_PROPERTY_PREFIX + "http.client.retryOnAllMethods";
   String ENCODE_URI_PARAMS_PROPERTY = SYSTEM_PROPERTY_PREFIX + "http.client.encodeUriParams";
+  String BASIC_LAX_DECODING_PROPERTY = SYSTEM_PROPERTY_PREFIX + "http.basic.laxDecoding";
```

Comment:
i'd prefer to rename this to `http.basic.laxBase64Decoding `, and the system property accordingly